### DISCUSSION TITLE
add SS-combo stat to score card

### DISCUSSION THREAD

**Discussion by tinDavid19428761** [ppy/osu#22723] [Ideas] [answered]
In the score card that shows after passing a map, I think it would be cool to see an 'SS'-combo along side the usual combo. This would show the longest streak a player hit combo without dropping accuracy. I think it would make score more interesting.

**Comment by mk56-spn** [comment, marked as answer]
Lazer sadly doesn't store per judgement accuracy for now ( and might not ever. Not sure what the direction on that was ) so this data would only be available on replays . 

Also feels a tad niche to be a metric shown but yeah

**Reply by tinDavid19428761** [reply]
As an outsider to the actual programming of lazer, thanks for explaining why this wouldn't be feasible. 👍
